extensions/wikia/Chat2/css/ChatEntryPoint.scss

```diff
@@ -64,6 +64,7 @@ body.skin-oasis .ChatMonobookEntryPoint {
 			font-size: 12px;
 			font-weight: normal;
 			line-height: 14px;
+			margin-top: 5px;
```

Comment:
![Why u no margin right and bottom?](https://i.chzbgr.com/maxW500/8501504256/hFE064191/)